sklearn/tree/tests/test_tree.py

```diff
@@ -1987,18 +1987,19 @@ def test_balance_property(criterion, Tree):
     assert np.sum(reg.predict(X)) == pytest.approx(np.sum(y))
 
 
-def test_poisson_zero_nodes():
+@pytest.mark.parametrize("seed", range(100))
```

Comment:
When this gets merged we should use less seeds, maybe `range(10)`?

Reply:
At my machine, the first _hit_ is with seed = 105.

Reply:
I got a first hit on seed = 31 :) Very low level stuff apparently.

Reply:
> When this gets merged we should use less seeds, maybe range(10)?

But I did not plan to merge such a large range. `range(3)` would be enough for the short term. In the future we might want to setup infra to do randomized pytest from a finite set of rng seeds (e.g. 0-999) but let's do that later :)